UnitTests/BraintreePayPalTests/BTPayPalRequest_Tests.swift

```diff
@@ -60,9 +64,12 @@ class BTPayPalRequest_Tests: XCTestCase {
         XCTAssertEqual(parameters["correlation_id"] as? String, "123-correlation-id")
         XCTAssertEqual(experienceProfile["address_override"] as? Bool, false)
         XCTAssertEqual(parameters["line_items"] as? [[String : String]], [["quantity" : "1",
-                                                                                   "unit_amount": "1",
-                                                                                   "name": "item",
-                                                                                   "kind": "credit"]])
+                                                                            "unit_amount": "1",
+                                                                            "name": "item",
+                                                                            "kind": "credit",
+                                                                            "upcCode": "upc-code",
+                                                                            "upcType": "UPC-A",
+                                                                            "imageUrl": "http://example/image.jpg"]])
```

Comment:
Not able run this test in local environment. Would like to know if will be tested by any existing CI jobs.